### PR TITLE
initial_tasks: make it possible to disable ansible

### PR DESCRIPTION
**Pull request by kshtsk**
Signed-off-by: Kyr Shatskyy <kyrylo.shatskyy@suse.com>

### CODE REVIEW

**Comment by kshtsk** [comment]
> This will make it impossible to run the core suites on openSUSE machines in the Sepia lab. . .

@smithfarm come again

**Comment by smithfarm** [comment]
Adding DNM for now (will change `ceph_cm: [STRING]` to `ceph_cm_ansible: [BOOLEAN]` which will hopefully be less confusing)

**Comment by kshtsk** [comment]
@smithfarm could you please review once again

File: teuthology/run.py [hunk 1]
```diff
@@ -226,9 +226,13 @@ def get_initial_tasks(lock, config, machine_type):
...
             {'selinux': None},
-            {'ansible.cephlab': None},
-            {'clock': None}
         ])
+    if config.get('ceph_cm_ansible', True):
+        init_tasks.append({'ansible.cephlab': None})
+
+    # sync_clock: 'clock' or 'clock.check'
+    sync_clock = config.get('sync_clock', 'clock')
+    init_tasks.append({sync_clock: None})
```
Comment (smithfarm) [inline teuthology/run.py]: Could the config option be called `sync_clock_task` instead of `sync_clock`?

Comment (kshtsk) [reply teuthology/run.py]: @smithfarm easy, check the next commit out

File: teuthology/run.py [hunk 2]
```diff
@@ -226,9 +226,13 @@ def get_initial_tasks(lock, config, machine_type):
...
             {'selinux': None},
-            {'ansible.cephlab': None},
-            {'clock': None}
         ])
+    if config.get('ceph_cm_ansible', True):
+        init_tasks.append({'ansible.cephlab': None})
+
+    # clock_sync_task: 'clock' or 'clock.check'
+    clock_sync_task = config.get('clock_sync_task', 'clock')
+    init_tasks.append({clock_sync_task: None})
```
Comment (zmc) [inline teuthology/run.py]: Is this an intentional change? I don't see it mentioned anywhere

Comment (kshtsk) [reply teuthology/run.py]: this is a collateral change and does not alter the behavior

**Pull request merged**